test/integration.js

```diff
@@ -365,7 +369,32 @@ test('ChatList methods', t => {
   t.end()
 })
 
-test('key transfer methods', t => {
+test.skip('key transfer', t => {
```

Comment:
In a test like this you should use `t.plan(N)` where `N` is the number of tests you expect (and remove `t.end()`).

Reply:
Didn't understand the plan method. What kind of iterations? Does it run the test multiple times or what's the reason of it?

Reply:
would like to skip, still don't really get it :D

Reply:
The idea is to ensure all sub tests are run before the whole test ends. In theory, this test _could_ end when `t.end()` is called, even though some of the other sub tests haven't been executed.